### ISSUE TITLE
Improve visual appearance of labels on metrics with grey background

### ISSUE THREAD

**Issue by eric-nieuwland** [ICTU/quality-time#5481]
**Is your feature request related to a problem? Please describe.**
I noticed in light mode the colours of labels and grey metrics are the same.
Unless one selects the metric - which darkens the grey - labels cannot be distinguished from the background.
[NOTE Entered as a feature request as this does not influence the proper operation]

**Describe the solution you'd like**
Use a darker grey for labels to create the visual inverse of the appearance in dark mode.

**Describe alternatives you've considered**
A thin outline for labels might be an alternative but would attract attention too much. 

**Additional context**
You may want to check https://en.wikipedia.org/wiki/Solarized for an approach to maintain visual similar contrasts in light mode and dark mode.

**Comment by fniessink** [comment]
This should be fixed by the migration to Material UI.
